README.md

```diff
@@ -1,3 +1,6 @@
+<div align="center">
+  <img height="500px" width="500px" src="https://opencv.org/wp-content/uploads/2020/07/OpenCV_logo_no_text_.png">
+</div>
```

Comment:
1. Markdown on GitHub has own support for images. No need HTML here
2. It looks like overkill: https://github.com/supersaint7780/opencv/tree/e6d07720f1fcfbc235e29e84319121c551db6b20 . There are no such details for using large images. Small image should be enough.
3. We don't need to fetch image from external link. Logo images are part of this repository: https://github.com/opencv/opencv/tree/4.x/doc

Reply:
thank you for your guidance😃